source/howto/CreateVMFromCommandLine.rst

```diff
@@ -18,100 +18,65 @@ Setting up the environment to select project
 
 After logging into a host, you have to set your environment variables.
 
-You can download a script to set these up for you from: https://openstack.stfc.ac.uk/project/api_access/openrc/. You will need to be signed in to the project you wish to use in the OpenStack web interface for this to work.
+The recommended method is downloading this script to set these up for you from: https://openstack.stfc.ac.uk/project/api_access/openrc/ (You will need to be signed in to the project you wish to use in the OpenStack web interface for this to work.)
 
 Or set them yourself in a script:-
 
 .. code-block:: bash
 
-  export OS_AUTH_URL=https://openstack.nubes.rl.ac.uk:5000/v3
+  export OS_AUTH_URL=https://openstack.stfc.ac.uk:5000/v3
+  export OS_PROJECT_ID=<project ID>
+  export OS_PROJECT_NAME=<project name>
+  export OS_PROJECT_DOMAIN_NAME=<domain name>
+  export OS_USER_DOMAIN_NAME=<user domain>
+  export OS_PROJECT_DOMAIN_ID=<project domain ID>
+  export OS_USERNAME=<fed ID>
+  export OS_PASSWORD=<password>
+  export OS_INTERFACE=public
   export OS_IDENTITY_API_VERSION=3
-  export OS_PASSWORD=************
-  export OS_PROJECT_DOMAIN_NAME=default
-  export OS_PROJECT_NAME="SCD Cloud Operations Team"
-  export OS_TENANT_NAME="SCD Cloud Operations Team"
-  export OS_USERNAME=<username>
-  export OS_USER_DOMAIN_NAME=<userdomain>
 
-Take note of the “PROJECT-NAME” and “TENANT-NAME” – they are the project that you wish to create the new VMs in.  The USER_DOMAIN_NAME and USERNAME are the user and the domain that user is in, that you are using to create the VMs. PASSWORD is your login to openstack – in clear text.
-It is common to put all of the above in a shell file (just as above, but with a file extension of .sh) and then “source” the environment variables so that they are part of your command shell So, if I put the above list of commands in a file “martin_openstack.sh” then I need to run “source ./martin_openstack.sh” after I have logged into a openstack server.
+Please take note of the required details from openstack by going to the desired project, and going into the submenu API_ACCESS (https://openstack.stfc.ac.uk/project/api_access/) and click on view credentials.
+
+.. csv-table:: variables
+  :header: "Parameter", "Description"
+
+  "OS_PROJECT_NAME", "they are the project that you wish to create the new VMs in."
+  "OS_USERNAME and OS_PROJECT_DOMAIN_NAME", "Are the username and the domain that user is in, which you are using to create the VM."
+  "OS_PASSWORD",  "Is your login to openstack (in plain text)." 
+
+It is common to put all of the above in a shell file (just as above, but with a file extension of .sh) and then “source” the environment variables so that they are part of your command shell.
```

Comment:
IMO we should drop this, we should just steer them to OpenRC files, as getting them to write their own is asking for trouble
If they want, they can always adapt an existing RC file

Reply:
I chatted to @jacob-ward in person, he's happy to drop this